content/recipes/prisma.md

```diff
@@ -516,6 +515,17 @@ This controller implements the following routes:
 
 - `/post/:id`: Delete a post by its `id`
 
+#### Issues with `enableShutdownHooks`
+
+Prisma interferes with NestJS `enableShutdownHooks` - Prisma listens for shutdown signals and will call process.exit() before your application shutdown hooks fire. To deal with this, you would need to add a listener for Prisma `beforeExit` event. 
+```typescript
+prismaService.$on('beforeExit', async () => {
+  logger.log('Prisma before exit...');
+  await app.close();
```

Comment:
Where do we do this that we have access to a service instance AND the app? This example could be more explicit (everything else in the recipe tells you what file it is dealing with)

Reply:
Hey @pjo336 - have a look now, does the added context help explain it more?

Reply:
That looks great! Thanks for the help!